packages/react/src/Accordion/Accordion.tsx

```diff
@@ -80,6 +80,7 @@ export const AccordionRoot = forwardRef<HTMLDetailsElement, AccordionRootProps>(
 
     useEffect(() => {
       const detailsElement = ref.current
+      /* istanbul ignore next */
```

Comment:
This seems tricky to test, are you happy for me to ignore this particular line?

Reply:
We have added this line on some places, but could you elaborate on why we need to add it here please? Haven't looked into it, so lacking context. It's an effect, so looks relatively important.

Reply:
> detailsElement (ref.current) is always going to be defined

I can't recall the exact reason anymore, but it would seem that this early return was very intentional 🤔 

The correct fix here would instead be to wrap all handlers inside an if statement, checking if both ref and ref.current are defined? That seems like a common use-case that we're handling elsewhere too?

Curious what you think. I'm fine ignoring coverage for this, as we have adequate coverage elsewhere now. Mostly wondering if there's any other ways to work around this and not ignore it entirely.

Reply:
Great, ty, will do